pyincore/utils/popdisloutputprocess.py

```diff
@@ -130,33 +132,39 @@ def pd_by_race(self, filename_json=None):
         huapd.loc[(huapd["hispan"] == 1) & huapd["dislocated"], "hud_re"] = "4"
         huapd.loc[(huapd["gqtype"] >= 1) & huapd["dislocated"], "hud_re"] = "5"
         hud_vals = huapd["hud_re"].value_counts()
+        # vacant HU do not dislocate
+        hud_vals[str(0)] = 0
         hua_disl = []
-        for i in range(len(race_categories)):
+        for i in range(len(race_categories) - 1):
             try:
                 hua_disl.append(int(hud_vals[str(i)]))
             except Exception:
                 hua_disl.append(0)
-        hua_disl.append(int(sum(hua_disl[1:])))
+        hua_disl.append(int(sum(hua_disl)))
 
         pd_disl = []
-        for i in range(len(race_categories)):
-            pd_disl.append(int(huapd["numprec"].where(huapd["hud_re"] == str(i)).sum()))
-        pd_disl.append(int(sum(pd_disl[1:])))
+        for i in range(len(race_categories) - 1):
+            # vacant HU do not dislocate
```

Comment:
this is not true.... i think they still dislocate. Otherwise the number doesn't add up

Reply:
I changed that but it does not make sense.